gemrb/plugins/GUIScript/GUIScript.cpp

```diff
@@ -1972,16 +1972,17 @@ static PyObject* GemRB_Window_HasControl(PyObject * /*self*/, PyObject* args)
 PyDoc_STRVAR( GemRB_Control_QueryText__doc,
 "===== Control_QueryText =====\n\
 \n\
-**Prototype:** GemRB.QueryText (WindowIndex, ControlIndex)\n\
+**Prototype:** GemRB.QueryText (WindowIndex, ControlIndex, UseSystemEncoding)\n\
 \n\
-**Metaclass Prototype:** QueryText ()\n\
+**Metaclass Prototype:** QueryText (UseSystemEncoding=False)\n\
```

Comment:
We currently use the above syntax, but this is fine, I guess.

Reply:
If you wish, I can change it, but yes, I think it should be OK.